packages/backend/src/services/action.ts

```diff
@@ -71,6 +71,10 @@ export const processAction = async (options: ProcessActionOptions) => {
     }
   } catch (error) {
     logger.error(error)
+    // log raw http error from StepError
+    if (error.cause) {
+      logger.error(error.cause)
+    }
```

Comment:
thinking we can be a bit more defensive (what if some of our dependencies start using `cause` as well and we forgot to wrap their errors in a `StepError`)

```suggestion
    logger.error(error)
    // log raw http error from StepError
    if (error instanceof StepError && error.cause) {
      logger.error(error.cause)
    }
```

Reply:
wah i did this initially but i thought it was an overkill omy